src/components/common/TmPage.vue

```diff
@@ -77,7 +77,7 @@ export default {
       default: false
     },
     error: {
-      type: Error,
+      type: [Error, Boolean],
```

Comment:
We have a rule about not allowing multiple types per property

Reply:
Good call. `TMPage` already treated it like a Boolean, so I've removed the `Error` type. Other values will be coerced to Boolean.